Changelog.md

```
@@ -15,9 +15,14 @@ This release gives some ❤️ to AvaloniaUI and other frameworks which use ILis
     * Add `IReadOnlyList<T>` interface to `EntityCollection` and `EntitySet` and `LoadResult`
     * Add `IList<T>` interface to `EntityCollection`
 * Nullability annotations for `EntitySet` and `EntityCollection`
+* Nullability annotations for a large part of the most used public API
+
+### Server
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)